scripts/ci/tags.yaml

```diff
@@ -138,6 +146,8 @@ ci_samples_wifi:
 
 ci_applications_matter: &ci_matter
   files:
```

Comment:
Shouldn't we update all of the tags (e.g.`ci_samples_wifi`, `ci_samples_openthread` and `ci_samples_nfc` are not updated) ?

Reply:
Intentionally not, I added them only for those big one (with a lot of number of tests).
There is another method to execute all tests (adding label for PR).

Reply:
Thanks for clarification